src/system/group.cr

```diff
@@ -0,0 +1,50 @@
+require "crystal/system/group"
+
+class System::Group
```

Comment:
This is missing docs describing what is a Group and how you generally use this class.